langchain4j-mcp/src/main/java/dev/langchain4j/mcp/client/DefaultMcpClient.java

```diff
@@ -32,12 +36,22 @@ public class DefaultMcpClient implements McpClient {
     private final String clientName;
     private final String clientVersion;
     private final String protocolVersion;
+    private final Duration toolExecutionTimeout;
+    private final JsonNode RESULT_TIMEOUT;
```

Comment:
Yea, but I initialize it in the constructor because I couldn't figure out a way with the fluent API to build such complex object without the last statement returning the whole object, not one nested field in it